MIT Press Books.js

```diff
@@ -0,0 +1,298 @@
+	{
+		"type": "web",
+		"url": "https://mitpress.mit.edu/books/series/adaptive-computation-and-machine-learning-series",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://mitpress.mit.edu/books/elements-causal-inference",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Elements of Causal Inference: Foundations and Learning Algorithms",
+				"creators": [
+					{
+						"firstName": "Jonas",
+						"lastName": "Peters",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Dominik",
+						"lastName": "Janzing",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Bernhard",
+						"lastName": "Schölkopf",
+						"creatorType": "author"
+					}
+				],
+				"date": "2017-11-29",
+				"ISBN": "9780262037310",
+				"language": "en",
+				"libraryCatalog": "MIT Press Books",
+				"numPages": "288",
+				"place": "Cambridge, MA, USA",
+				"publisher": "MIT Press",
+				"series": "Adaptive Computation and Machine Learning series",
+				"shortTitle": "Elements of Causal Inference",
+				"attachments": [
+					{
+						"title": "Full Text PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://mitpress.mit.edu/search?keywords=deep+learning",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://mitpress.mit.edu/contributors/ian-goodfellow",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://mitpress.mit.edu/books/sciences-artificial-reissue-third-edition-new-introduction-john-laird",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "The Sciences of the Artificial, Reissue Of The Third Edition With A New Introduction By John Laird",
+				"creators": [
+					{
+						"firstName": "Herbert A.",
+						"lastName": "Simon",
+						"creatorType": "author"
+					}
+				],
+				"date": "2019-08-13",
+				"ISBN": "9780262537537",
+				"language": "en",
+				"libraryCatalog": "MIT Press Books",
+				"numPages": "256",
+				"place": "Cambridge, MA, USA",
+				"publisher": "MIT Press",
+				"attachments": [],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://mitpress.mit.edu/books/construction-site-possible-worlds",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Construction Site for Possible Worlds",
+				"creators": [
+					{
+						"firstName": "Amanda",
+						"lastName": "Beech",
+						"creatorType": "editor"
+					},
+					{
+						"firstName": "Robin",
+						"lastName": "Mackay",
+						"creatorType": "editor"
+					},
+					{
+						"firstName": "James",
+						"lastName": "Wiltgen",
+						"creatorType": "editor"
+					}
+				],
+				"date": "2020-05-12",
+				"ISBN": "9781913029579",
+				"language": "en",
+				"libraryCatalog": "MIT Press Books",
+				"numPages": "136",
+				"place": "Cambridge, MA, USA",
+				"publisher": "MIT Press",
+				"series": "Urbanomic",
```

Comment:
Actually, this looks like we should better save this as the publisher info, cf. https://www.urbanomic.com/book/construction-site-possible-worlds/ .